tasks/app/config.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Application Users management related tasks for Invoke.
```

Comment:
minor but this is not users management, I suspect copy & tweak

Reply:
Fixed